runtime/src/accounts.rs

```diff
@@ -428,9 +428,32 @@ impl Accounts {
         accounts_balances
     }
 
+    pub fn calculate_capitalization(&self, ancestors: &Ancestors) -> u64 {
+        let balances =
+            self.load_by_program(ancestors, None)
```

Comment:
Would like to move away from `load_by_program` with a `None` argument, this is kind of just an obfuscated `load_all` kind of function, right?

Reply:
I neglected the confusing name... not anymore!: 21d6864ccf395ff86c510df2823277078d6dc20c